### ISSUE TITLE
Alertmanager 309 response code in alert manger 

### ISSUE THREAD

**Issue by om3171991** [prometheus/alertmanager#2691]
Hey Team, 

I am trying to configure opsgenie with below config but getting 309 error with error message as 

ErrorMessage:**level=error ts=2021-09-03T13:11:41.023Z caller=dispatch.go:309 component=dispatcher msg="Notify for alerts failed" num_alerts=87 err="email-alert/opsgenie[0]: notify retry canceled due to unrecoverable error after 1 attempts: templating error: template: :1:195: executing \"\" at <.GroupLabels.severity>: severity is not a method but has arguments"**

Config:

**global: {}

route:
  group_by:
  - alertname
  - cluster
  - service
  - severity
  group_wait: 10s
  group_interval: 10s
  repeat_interval: 1h
  receiver: 'email-alert'

receivers:
- name: opsgenie-alert
  opsgenie_configs:
  - api_key: <api_key>
    api_url: https://api.opsgenie.com
    priority: '{{ if or (eq .GroupLabels.severity "critical") (eq .GroupLabels.severity "high")}}P1{{else if or (eq .GroupLabels.severity "warning") (eq .GroupLabels.severity "medium")}}P2{{else if .GroupLabels.severity "low" }}P3{{else}}P4{{end}}'
    message: '{{- range .Alerts }} {{ if .Labels.cluster }}{{- .Labels.cluster}}|{{end}}{{ if .Labels.service }}{{- .Labels.service}}|{{end}}{{ if .Labels.alertname }}{{- .Labels.alertname}}{{end}}{{end}}'
    description: '{{- range .Alerts }} {{- range $key, $value := .Annotations }} {{- println $key ":" $value}} {{- end}} {{- end}}'
    send_resolved: false**

Basically, there are few alerts with send severity as [high,medium,low] and few are sending like [critical,warning,info,none]. and merging these to using OR helm method.

**Comment by TheMeier** [comment]
This is a templating error. Also 309 is not a response code, it is the line ind dispatch.go where the error was raised.

If you believe this issue is still relevant, please feel free to reopen it or create a new one.
For further support please use the [community support](https://prometheus.io/community/) channels.
